_unittest/test_41_3dlayout_modeler.py

```diff
@@ -509,7 +509,7 @@ def test_19A_analyze_setup(self):
     @pytest.mark.skipif(is_ironpython, reason="Not supported with IronPython")
     def test_19B_analyze_setup(self):
         assert self.solve.mesh.generate_mesh("Setup1")
-        assert self.solve.analyze_setup("Setup1")
+        assert self.solve.analyze_setup("Setup1", num_cores=1)
```

Comment:
Could we raise the number of cores to 2 to increase the test speed even more?

Reply:
ideally yes. The problem is that build machines has only 8 cores and we use 8 concurrent pytest. I'm worried about some bottlenecks